### DISCUSSION TITLE
Improve finding this package at PyPI

### DISCUSSION THREAD

**Discussion by PanderMusubi** [helloflask/bootstrap-flask#334] [General]
Please upvote https://github.com/pypi/warehouse/issues/10718 👍 for improving finding this package at PyPI. See also https://github.com/pypi/warehouse/issues/14738#issuecomment-1759670231

**Comment by savchenko** [comment]
Spam?.. Ping @greyli

**Reply by greyli** [reply]
Nope, PanderMusubi is one of the maintainers. But I think this discussion can be closed now.
